### PR TITLE
docs: update changelog for 0.22 release

### PR DESCRIPTION
**Pull request by mjsax**
### Description 
Update changelog for 0.22 release

### Reviewer checklist
- [ ] Ensure docs are updated if necessary. (eg. if a user visible feature is being added or changed).
- [ ] Ensure relevant issues are linked (description should include text like "Fixes #<issue number>")

### CODE REVIEW

**Review by JimGalasyn — APPROVED** [review]
LGTM!

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,69 @@
...
+* Allow scalable push queries to handle rebalances ([#7988](https://github.com/confluentinc/ksql/pull/7988)) ([b3dbed3](https://github.com/confluentinc/ksql/commit/b3dbed35936684170355588e358d7a2d25692ed5))
+* create and write a query plan for source tables ([#8043](https://github.com/confluentinc/ksql/pull/8043)) ([381b7bf](https://github.com/confluentinc/ksql/commit/381b7bf6c943b11136cb0e952109b77c4b95653c))
+* fail scalable push query if error is detected in subscribed persistent query ([#7996](https://github.com/confluentinc/ksql/pull/7996)) ([eed501c](https://github.com/confluentinc/ksql/commit/eed501cb5611868c2546dda56c196a8608009b7e))
+* perform SchemaRegistry permissions on C*AS sink subjects ([#8039](https://github.com/confluentinc/ksql/pull/8039)) ([6878233](https://github.com/confluentinc/ksql/commit/6878233418157b8bddd27c76a9c8fe88aaa5feba))
+* Propose KLIP 53: Pull Queries on Streams ([#7983](https://github.com/confluentinc/ksql/pull/7983)) ([58ae6a2](https://github.com/confluentinc/ksql/commit/58ae6a24e261ce1bb4393a92a725a6c442e46364))
```
Comment (jzaralim) [inline CHANGELOG.md:34]: This is documentation
```suggestion

```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,69 @@
 # Change Log
 
+## [0.22.0](https://github.com/confluentinc/ksql/releases/tag/v0.22.0-ksqldb) (2021-11-03)
+
+### Features
+
+* add configurations around endpoint logging ([#8249](https://github.com/confluentinc/ksql/pull/8249)) ([21f4e03](https://github.com/confluentinc/ksql/commit/21f4e039c4fe0a4206fbd0c1b7c82342cf27c3fe))
+* add consumerGroupId to QueryDescription ([#8073](https://github.com/confluentinc/ksql/pull/8073)) ([cce585b](https://github.com/confluentinc/ksql/commit/cce585b0991a5a7fa39d3e24f327fdd84783f2ff))
+* add CumulativeSum total bytes metric ([#7987](https://github.com/confluentinc/ksql/pull/7987)) ([aa213bd](https://github.com/confluentinc/ksql/commit/aa213bd3153cc8d9d9d5b218210dfb6b5806948d))
+* add feature flag to disable source table materialization ([#8085](https://github.com/confluentinc/ksql/pull/8085)) ([5416cde](https://github.com/confluentinc/ksql/commit/5416cde808dea59e3091eb637ab733723ff9bdcc))
```
Comment (jzaralim) [inline CHANGELOG.md:10]: All the source table commits can be merged into one, https://github.com/confluentinc/ksql/blob/73ebeab02bad94c454b538a8c1c605cd35f89709/CHANGELOG.md?plain=1#L8 for example

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,69 @@
...
+* issue 7948. Allow insert into table using Java API ([#8114](https://github.com/confluentinc/ksql/pull/8114)) ([a6c2cac](https://github.com/confluentinc/ksql/commit/a6c2cac8cca9a1ca1194f1fb09a0cf7a99ae47ea))
+* Removes config check to insert SPQ Processor ([#8062](https://github.com/confluentinc/ksql/pull/8062)) ([a0be1d5](https://github.com/confluentinc/ksql/commit/a0be1d582b5cc27d01de98ab745d0559e9ad5ff9))
+* skip adding invalid if not exists to cmd topic ([#8206](https://github.com/confluentinc/ksql/pull/8206)) ([e164b18](https://github.com/confluentinc/ksql/commit/e164b186f74ce4b0a450be9a8ba88d5921bc2205))
+* stream pull query internal overrides shouldn't clash with query configs ([#8166](https://github.com/confluentinc/ksql/pull/8166)) ([0c57258](https://github.com/confluentinc/ksql/commit/0c57258bd6b19765a3d40173ffba6f21d864e50f))
+* update title in KLIP-54 ([#8099](https://github.com/confluentinc/ksql/pull/8099)) ([d22d4bc](https://github.com/confluentinc/ksql/commit/d22d4bc4b1a7f53809ba9f2a550ddb68ca7bfeb3))
```
Comment (jzaralim) [inline CHANGELOG.md:60]: Documentation
```suggestion

```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,69 @@
...
+* use a looser check on the error msg for overflow ([#8098](https://github.com/confluentinc/ksql/pull/8098)) ([0088fb1](https://github.com/confluentinc/ksql/commit/0088fb1b22b0abc929c14f856dd58494372137a7))
+* date parsing functions: case-insensitive name parsing ([#8015](https://github.com/confluentinc/ksql/pull/8015)) ([73bdb18](https://github.com/confluentinc/ksql/commit/73bdb184c1f225030577bb7771a55ab95ce73cb8))
+* Fixes pull query latency distribution metrics ([#7992](https://github.com/confluentinc/ksql/pull/7992)) ([c798cd7](https://github.com/confluentinc/ksql/commit/c798cd76f4d4888c959bdec0a0b3c8c11fdc5dd9))
+* Lower session timeout for RQTT ([#8053](https://github.com/confluentinc/ksql/pull/8053)) ([937ef57](https://github.com/confluentinc/ksql/commit/937ef57a7db0a70b961d0a5bb70f6329bb781f7e))
```
Comment (jzaralim) [inline CHANGELOG.md:64]: Test
```suggestion

```

**Comment by JimGalasyn** [comment]
@mjsax Possibly target this PR to the `0.22.x-ksqldb` branch? (ASsuming that you'll cut `0.22.0-ksqldb` from there.)

**Comment by mjsax** [comment]
Closing in favor of https://github.com/confluentinc/ksql/pull/8375

**Pull request closed without merging**